source/_components/image_processing.opencv.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "OpenCV"
+description: "Instructions how to integrate OpenCV image processing into Home Assistant."
+date: 2017-01-25 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: OpenCV_Logo.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.38
+---
+
+The `opencv` image processing platform allows you to create a standalone image processor without the linked camera entity as mentioned in the [OpenCV page](https://home-assistant.io/components/opencv).
+
+### {% linkable_title Configuration Home Assistant %}
+
+```yaml
+# Example configuration.yaml entry
+opencv:
+  - name: Detect Face
+    entity_id:
+      - camera.front_door
+      - camera.living_room
+    classifier:
+      - file_path: /path/to/classifier/face.xml
+        name: face
+      - file_path: /path/to/classifier/face_profile.xml
```

Comment:
Let's say you want to match "Family" and you have a classifier for "Alok" (alok_face.xml), "Wife" (alok_wife_face.xml), "Kiddo" (alok_kiddo_face.xml).

This allows you to have:
```
classifier_group:
  name: 'Family'
  classifier:
    - name: 'Alok'
      file_path: 'alok_face.xml'
    - name: 'Wife'
      file_path: 'alok_wife_face.xml'
    - name: 'Kiddo'
      file_path: 'alok_kiddo_face.xml'
```

The matches would be:

```
Family: {
  Alok: {
    (region)
  },
  Wife: {
    (region)
  },
  Kiddo: {
    (region)
  }
}
```